addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -55,6 +55,17 @@ def get_mail_values(self, res_ids):
                             r'<html\1 xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">',
                             str(body)
                         )
+                        complete_body_html = complete_body_html.replace(
+                            '</head>',
+                            """<!--[if mso]>
+                                <xml>
+                                    <o:OfficeDocumentSettings>
+                                        <o:AllowPNG/>
+                                        <o:PixelsPerInch>96</o:PixelsPerInch>
```

Comment:
I think you had a source for this, right ? Because 96 feels kind of random without citing a source for it.

Reply:
There's many sources on this, it's apparently a standard thing to put in email html. [Here](https://www.htmlemailcheck.com/knowledge-base/dpi-scaling-in-outlook-2007-2013/)'s one of them.

Reply:
Done adding a comment with that link.